src/Okta.Sdk/DefaultRetryStrategy.cs

```diff
@@ -0,0 +1,158 @@
+﻿// <copyright file="DefaultRetryStrategy.cs" company="Okta, Inc">
+// Copyright (c) 2014 - present Okta, Inc. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Okta.Sdk
+{
+    /// <summary>
+    /// Default retry strategy
+    /// </summary>
+    public class DefaultRetryStrategy : IRetryStrategy
+    {
+        /// <summary>
+        /// The default delta used in the back-off formula to account for some clock skew in our service
+        /// </summary>
+        public const int DefaultBackOffSecondsDelta = 1;
+
+        private readonly int _maxRetries;
+        private readonly int _requestTimeout;
+        private readonly int _backoffSecondsDelta;
+
+        // Now we are only managing 429 errors, but we can accept other codes in the future
+        private IList<HttpStatusCode> _retryableStatusCodes = new List<HttpStatusCode> { (HttpStatusCode)429 };
```

Comment:
Just a suggestion, but maybe the status codes should be settable in the constructor, with 429 being the default value (just like `backoffSecondsDelta`).

Reply:
As we already discussed,  now we only want to support 429.